lib/new_relic/agent/agent.rb

```diff
@@ -9,6 +9,7 @@
 require 'zlib'
 require 'stringio'
 require 'new_relic/constants'
+require 'new_relic/traced_thread'
```

Comment:
From looking at the existing "constants" and "coerce" it looks like we were already out of alphabetical order here. Is there any sort order that we follow for the requires here?

Reply:
I'm not aware of a sort order, but we could create a new standard?

Reply:
Yeah it didn't look like anything was trying to be in alphabetical order at all, I just put this one up here instead of lower in the list since this one was just a `new_relic/something` rather than a lot of the other ones that are `new_relic/agent/something`.